spiceypy/tests/test_spiceerrors.py

```diff
@@ -43,7 +43,25 @@ def test_get_spicey_exception():
     spice.reset()
 
 
-def test_empty_kernel_pool_exception():
+def test_getSpiceyPyError():
+    with pytest.raises(spice.stypes.SpiceyPyError):
+        spice.furnsh(os.path.join(cwd, "_null_kernel.txt"))
+    spice.reset()
+
+
+def test_getSpiceyPyIOError():
+    with pytest.raises(spice.support_types.SpiceyPyIOError):
+        spice.furnsh(os.path.join(cwd, "_null_kernel.txt"))
+    spice.reset()
+
+
+def test_getIOError():
+    with pytest.raises(IOError):
+        spice.furnsh(os.path.join(cwd, "_null_kernel.txt"))
+    spice.reset()
```

Comment:
These look like they're testing the exact same thing giving different results. Perhaps a comment to explain the differences?